### PR TITLE
Update jib to 2.5.x

### PR DESCRIPTION
**Pull request by mpeddada1**
Gradle 2.5.0
Maven 2.5.2 (released on Aug 11)

### CODE REVIEW

**Comment by chanseokoh** [comment]
@GoogleContainerTools/skaffold-team is it expected that the two "integration partition 3" tests fail due to the survey message thing? I see all recent PRs fail for the same reason.

```
=== Failed Tests ===
/integration/TestHouseKeepingMessagesShownForDev
time="2020-08-07T21:29:45Z" level=info msg="[skaffold run --default-repo gcr.io/k8s-skaffold -c /tmp/config294819199]"
time="2020-08-07T21:29:46Z" level=info msg="Ran in 1.245899625s"
    TestHouseKeepingMessagesShownForDev: housekeeing_messages_test.go:37: expected output Help improve Skaffold! not found in output: Generating tags...
         - skaffold-example -> gcr.io/k8s-skaffold/skaffold-example:v1.13.1-7-ge778ef8
        Checking cache...
         - skaffold-example: Found Locally
        Tags used in deployment:
         - skaffold-example -> gcr.io/k8s-skaffold/skaffold-example:5d936e7114043417ddf6fad6c6879ba62ff2ddb3939e3ae1a81d97f3a7932302
        Loading images into kind cluster nodes...
         - gcr.io/k8s-skaffold/skaffold-example:5d936e7114043417ddf6fad6c6879ba62ff2ddb3939e3ae1a81d97f3a7932302 -> Found
        Images loaded in 147.205733ms
        Starting deploy...
         - pod/getting-started created
        Waiting for deployments to stabilize...
        Deployments stabilized in 17.702296ms
        You can also run [skaffold run --tail] to get the logs
--- FAIL: TestHouseKeepingMessagesShownForDev (1.25s)
```

How can we proceed?

File: examples/jib-multimodule/pom.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
   <properties>
     <java.version>1.8</java.version>
-    <jib.maven-plugin-version>2.4.0</jib.maven-plugin-version>
+    <jib.maven-plugin-version>2.5.1</jib.maven-plugin-version>
```
Comment (chanseokoh) [inline examples/jib-multimodule/pom.xml:21]: 2.5.2 for Maven

**Comment by loosebazooka** [comment]
So I think when I updated 2.4.0, I did it wrong (https://github.com/GoogleContainerTools/skaffold/pull/4308)
We're only supposed to update `integration/examples` and `integration/testdata`, all the examples from `integration/examples` get copied to `examples` when a skaffold release is made (this ensure compatibility with the skaffold api).

**Comment by loosebazooka** [comment]
other files to update:
- `integration/testdata/debug/java/pom.xml`
- `integration/examples/jib-sync/pom.xml`

**Comment by briandealwis** [comment]
It's ok to update `examples/` with `integration/examples` as long as they're compatible with the current Skaffold release.  We never `examples/` alone as they'll be overwritten on the next release.

File: examples/jib-multimodule/pom.xml [hunk 2]
```diff
@@ -18,7 +18,7 @@
 
   <properties>
     <java.version>1.8</java.version>
-    <jib.maven-plugin-version>2.4.0</jib.maven-plugin-version>
+    <jib.maven-plugin-version>2.5.0</jib.maven-plugin-version>
```
Comment (chanseokoh) [inline examples/jib-multimodule/pom.xml:21]: 2.5.2

**Pull request merged**